lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -46,13 +46,7 @@
         </button>
     <% } else {%>
         <div class="no-action-message">
-            <%- gettext('Coming Soon') %>
-        </div>
-        <div class="enrollment-opens">
-            <%- gettext('Enrollment Opens on') %>
-            <span class="enrollment-open-date">
-                <%- enrollment_open_date %>
-            </span>
+            <%- gettext('Not Currently Available') %>
```

Comment:
I've confirmed with Lauren that in light of the above, there's currently no use case for the "Coming Soon" message. We do need the "Not Currently Available" message to cover cases in which all course runs have ended. Let me know if you have questions. I'll otherwise be leaving this PR in its current state.